eng/Versions.props

```diff
@@ -43,7 +44,7 @@
     <MicrosoftDotNetRemoteExecutorVersion>5.0.0-beta.20063.2</MicrosoftDotNetRemoteExecutorVersion>
     <MicrosoftDotNetVersionToolsTasksVersion>5.0.0-beta.20063.2</MicrosoftDotNetVersionToolsTasksVersion>
     <!-- Installer dependencies -->
-    <MicrosoftNETCoreAppVersion>5.0.0-alpha.1.19562.8</MicrosoftNETCoreAppVersion>
+    <MicrosoftNETCoreAppVersion>5.0.0-alpha.1.20071.1</MicrosoftNETCoreAppVersion>
```

Comment:
I'm concerned that this version is being used... the installer tests need to be running on the current build's outputs or else they aren't covering anything. 😕 Can you link the issue you mentioned you'd file for this?

Reply:
I'm surprised the installer tests use this property, I thought this property was used by package testing only. 

I mentioned it here:
https://github.com/dotnet/runtime/issues/1823#issuecomment-577313221

Could you elaborate how the installer tests use this instead of the recently built product?

Reply:
> Could you elaborate how the installer tests use this instead of the recently built product?

No, I'm not familiar with it and I'm surprised by it when you mentioned you had to change this to make the installer tests work.

Reply:
I didn't mean installer tests, I mean libraries all configuration package tests... we run some tests on our oob packages to make sure closure is complete and that we don't ship duplicate types and that all supported rids for those packages publish correctly, etc.

Reply:
Ah. This should be moved out from `<!-- Installer dependencies -->` then, it seems to me. Thanks for the background.

Reply:
I see, that was a result from the repo merge as in corefx it was "core-setup dependencies" maybe when we consolidated it was just a find and replace from "core-setup" to "installer". Will remove.

Reply:
Actually, what it means is that those dependencies as built by the installer. If you look the file, those comments split the sections on where are those packages coming from, for example look at runtime-assets dependencies which comes from https://github.com/dotnet/runtime-assets or arcade dependencies.

Reply:
Oh, that makes sense. Sorry for the confusion on this, no need to change it.